.vscode/settings.json

```diff
@@ -1,7 +1,7 @@
 {
   "editor.codeActionsOnSave": {
-    "source.fixAll": true,
-    "source.organizeImports": true
+    "source.fixAll": "explicit",
+    "source.organizeImports": "explicit"
```

Comment:
what a change bro 😎

Reply:
breaking change

Reply:
rip our beloved api, press F to pay respects